tickets/migrations/0005_auto_20180608_1554.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('tickets', '0004_auto_20161024_1658'),
+        ('tickets', '0005_auto_20171031_2322'),
```

Comment:
This looks suspicious... why is an old migration being modified?

Reply:
Master had a migration after this branch was started locally which was causing travis to fail

Reply:
Problem is, 0005_auto_20171031_2322 doesn't exist anywhere... as far as I can see...

Reply:
Is part of this PR